src/main/java/cs/f10/t1/nursetraverse/logic/commands/appointment/EditAppointmentCommand.java

```diff
@@ -41,20 +31,20 @@
             + "by the index number used in the displayed appointment list. "
             + "Existing values will be overwritten by the input values.\n"
             + "Parameters: INDEX (must be a positive integer) "
-            + "[" + PREFIX_APPOINTMENT_START_DATE_AND_TIME + "APPOINTMENT_START_DATE_AND_TIME] "
-            + "[" + PREFIX_APPOINTMENT_END_DATE_AND_TIME + "APPOINTMENT_END_DATE_AND_TIME] "
-            + "[" + PREFIX_PATIENT_INDEX + "APPOINTMENT_END_DATE_AND_TIME] "
-            + "[" + PREFIX_RECUR_YEARS + "YEARS] "
-            + "[" + PREFIX_RECUR_MONTHS + "MONTHS] "
-            + "[" + PREFIX_RECUR_WEEKS + "WEEKS] "
-            + "[" + PREFIX_RECUR_DAYS + "DAYS] "
-            + "[" + PREFIX_RECUR_HOURS + "HOURS] "
-            + "[" + PREFIX_RECUR_MINUTES + "MINUTES] "
-            + "[" + PREFIX_APPOINTMENT_DESCRIPTION + "DESCRIPTION] "
+            + "[" + CliSyntax.PREFIX_APPOINTMENT_START_DATE_AND_TIME + "APPOINTMENT_START_DATE_AND_TIME] "
+            + "[" + CliSyntax.PREFIX_APPOINTMENT_END_DATE_AND_TIME + "APPOINTMENT_END_DATE_AND_TIME] "
+            + "[" + CliSyntax.PREFIX_PATIENT_INDEX + "APPOINTMENT_END_DATE_AND_TIME] "
+            + "[" + CliSyntax.PREFIX_RECUR_YEARS + "YEARS] "
+            + "[" + CliSyntax.PREFIX_RECUR_MONTHS + "MONTHS] "
+            + "[" + CliSyntax.PREFIX_RECUR_WEEKS + "WEEKS] "
+            + "[" + CliSyntax.PREFIX_RECUR_DAYS + "DAYS] "
+            + "[" + CliSyntax.PREFIX_RECUR_HOURS + "HOURS] "
+            + "[" + CliSyntax.PREFIX_RECUR_MINUTES + "MINUTES] "
+            + "[" + CliSyntax.PREFIX_APPOINTMENT_DESCRIPTION + "DESCRIPTION] "
             + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_APPOINTMENT_START_DATE_AND_TIME + "01-11-2019 1000 "
-            + PREFIX_RECUR_MONTHS + "4 "
-            + PREFIX_APPOINTMENT_DESCRIPTION + "Dental and health checkup";
+            + CliSyntax.PREFIX_APPOINTMENT_START_DATE_AND_TIME + "01-11-2019 1000 "
+            + CliSyntax.PREFIX_RECUR_MONTHS + "4 "
+            + CliSyntax.PREFIX_APPOINTMENT_DESCRIPTION + "Dental and health checkup";
```

Comment:
Same issue as mentioned before, I think the renaming probably messed this up somehow, but we can just import these one by one above from `CliSyntax` so that the constant we are using can be directly referenced to.